tests/test_ontology.py

```diff
@@ -36,5 +47,28 @@ def test_nbadapter_repo_annotations(test_notebook_repo):
     assert (oda["1000integer_15integer_emax_keV_lower_limit_upper_limit"], oda["lower_limit"], rdflib.Literal(15)) in G
 
     
+def test_nb2rdf(test_notebook_repo):
+    from nb2workflow.nbadapter import NotebookAdapter, find_notebooks
+    from nb2workflow import ontology
+
+    nbas = find_notebooks(test_notebook_repo)
+
+    G = rdflib.Graph()
+
+    for target, nba in nbas.items():
+        rdf_nb = ontology.nb2rdf(nba.notebook_fn)
+        G.parse(data=rdf_nb)
+        G.bind("oda", oda)
+        G.bind('rdfs', rdfs)
+        wfl_p_ns = rdflib.Namespace(wfl_p_ns_str.format(workflow_name=nba.unique_name))
+        assert (wfl_p_ns["scwid"], oda["default_value"], rdflib.Literal("066500110010.001")) in G
+        assert (wfl_p_ns["scwid"], rdf_type, oda_integral["ScWID"]) in G
+        assert (wfl_p_ns["nbins"], oda["default_value"], rdflib.Literal(100)) in G
```

Comment:
For input notebook, `default_value` is the default value. If extracted from the output one, it's the value actually used. So probably it could be just "value"?

Reply:
Good point, thanks! Whereas it should be `default_value` if it's a input notebook? Or we can just leave `value` to keep things easy...

Reply:
`value` sounds ok. It may have different meaning, different sort of value